migration/migration-technical-details.md

```diff
@@ -51,12 +55,25 @@ After the pre-staking phase, a clear overview of the registered validators and t
 
 ## Activation Phase
 
-Only pre-registered validators can participate in this phase. The activation phase serves two primary purposes:
+Only pre-registered validators can participate in this phase. The activation phase serves the purpose of executing the transition. At the first candidate block's height (block 3 456 000), if at least 80% of the total stake signals readiness for the transition, the candidate block becomes the transition block. A special activation tool captures and migrates the state of the PoW chain at the transition block, including the transaction history, to generate the "genesis block" for the new PoS chain.
+
+The Activation Phase follows a specific sequence of events:
+1. The candidate block is mined at block height 3 456 000, which opens the first activation window.
+2. The network waits for 10 confirmations to prevent block reversion.
+3. Once 10 confirmation blocks are mined, registered validators capture the state of block 3 456 000 and start generating the "genesis" block.
```

Comment:
3. After 10 block confirmations, registered validators capture the state at block 3 456 000 and start generating the PoS "genesis" block